src/Dialogs/LockScreen.cpp

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright_License {
+
+  XCSoar Glide Computer - http://www.xcsoar.org/
+  Copyright (C) 2000-2016 The XCSoar Project
+  A detailed list of copyright holders can be found in the file "AUTHORS".
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+}
+*/
+
+#include "Dialogs/LockScreen.hpp"
+//#include "Language/Language.hpp"
+#include "Form/Button.hpp"
+#include "Form/Form.hpp"
+#include "Form/Frame.hpp"
+#include "Look/DialogLook.hpp"
+#include "Screen/Layout.hpp"
+#include "Screen/SingleWindow.hpp"
+#include "Util/StaticArray.hxx"
+#include "UIGlobals.hpp"
+
+#include <assert.h>
+
+void
+ShowLockBox()
+{
+  SingleWindow &main_window = UIGlobals::GetMainWindow();
+
+  const unsigned button_height = Layout::GetMinimumControlHeight();
+  const unsigned button_width = button_height;
+
+  // Create dialog
```

Comment:
Writing code comments is good, but only if they add value, if they explain something that is not already obvious. Code comments which explain something extremely trivial and obvious don't add value; they just distract from reading the code. This is one such.